net/tstun/wrap.go

```diff
@@ -88,7 +85,7 @@ type Wrapper struct {
 	//
 	// Empty reads are skipped by Wireguard, so it is always legal
 	// to discard an empty packet instead of sending it through t.outbound.
-	outbound chan []byte
+	outbound chan tunReadResult
```

Comment:
document whether/when this channel is closed. (never?)

Reply:
Done; same for bufferConsumed.